activejob-uniqueness.gemspec

```diff
@@ -30,11 +30,4 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'activejob', '>= 4.2', '< 7.1'
   spec.add_dependency 'redlock', '>= 1.2', '< 3'
-
-  spec.add_development_dependency 'appraisal', '~> 2.3.0'
-  spec.add_development_dependency 'bundler', '>= 2.0'
-  spec.add_development_dependency 'pry-byebug', '> 3.6.0'
-  spec.add_development_dependency 'rspec', '~> 3.0'
-  spec.add_development_dependency 'rubocop', '~> 1.28'
-  spec.add_development_dependency 'rubocop-rspec', '~> 2.10'
```

Comment:
Hey @laurafeier 
Thank you for the PR!
Could you explain why this change is necessary?

Reply:
Ah, I see `Gemspec/DevelopmentDependencies: Specify development dependencies in Gemfile`
https://docs.rubocop.org/rubocop/cops_gemspec.html#gemspecdevelopmentdependencies

TBH I do not see any value in this change. I'd prefer to disable the cop instead

Reply:
This cop is not available for older versions of rubocop which are installed with older version. If you disable it, the rule is not recognized by older rubocop and it fails. I think this is the only way to have it working with older & newer ruby versions.

Reply:
Brilliant! Thank you